lib/connect_proto/src/medication_administration.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package primary.connect;
+option go_package = "primary.connect";
+
+import "google/protobuf/timestamp.proto";
+
+import "coded_value.proto";
+import "identifier.proto";
+
+message MedicationAdministration {
+  message Medication { // What is administered
+    string lot_number = 1; // Lot number product component
+    google.protobuf.Timestamp expiration_date = 2; // Product expiration date
+    CodedValue manufacturer = 3; // Code for the manufacturer
+  }
+
+  message Dosage { // Details of how medication is taken
+    CodedValue site = 1;  // Medication administration site
+    CodedValue route = 2; // Medication administration route
+    CodedValue method = 3;  // Medication administration method
+    string dose = 4; // Amount of medication per dose
+    CodedValue units = 5; // Medication units of measurement
+    string series_number = 6; // Dose number administered within series_recommended
+    string series_recommended = 7; // Number of recommended doses in a prescribed series
+  }
+
+  message AdministeringProvider { // The administering providers details
+    string first_name = 1; // first name of the administering provider
+    string last_name = 2; // last name of the administering provider
+    string degree = 3; // the administering provider's degree
+  }
```

Comment:
Don't need this. We already have a [`Provider`](https://github.com/PrimaryDotHealth/connect-proto/blob/main/lib/connect_proto/src/provider.proto)
```suggestion
```